store/modules/playback/actions.js

```diff
@@ -217,13 +220,11 @@ export default {
 
     try {
       if (no_barcode == true) {
-        result = await Vue.axios.get(
-          `${baseurl}/${endpoint}?time_index=${time_index}`
-        );
+        const whole_url_no_barcode = `${baseurl}/${endpoint}?time_index=${time_index}`;
+        result = call_axios_get_from_vuex(whole_url_no_barcode, context);
       } else {
-        result = await Vue.axios.get(
-          `${baseurl}/${endpoint}?time_index=${time_index}&barcode=${barcode}&is_hardware_test_recording=${payload.is_hardware_test_recording}`
-        );
+        const whole_url = `${baseurl}/${endpoint}?time_index=${time_index}&barcode=${barcode}&is_hardware_test_recording=${payload.is_hardware_test_recording}`;
+        result = call_axios_get_from_vuex(whole_url, context);
       }
     } catch (error) {
       if (result.status != 200) {
```

Comment:
it looks like this catch isn't ever hit during testing because a try/catch is already inside the new `call_axios_get_from_vuex` function. please remove it from this file

Reply:
Agreed, will remove the try/catch